source/install/install-rhel-66-mysql.rst

```diff
@@ -0,0 +1,66 @@
+..  _install-rhel-66-mysql:
+
+Installing MySQL Database Server
+================================
+
+Install and set up the database for use by the Mattermost server. You can install either MySQL or PostgreSQL. To install PostgreSQL, see :ref:`install-rhel-66-postgresql`.
```

Comment:
I'm wondering if we should have a section for "Installing Database Server", then a subsection for MySQL and PostgreSQL each.

Scanning through the doc and/or TOC, it appears as though you need to set up both, but in fact it's just setting up one database server, either MySQL or PostgreSQL.